src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +20,15 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        Supplier<RuntimeException> exceptionSupplier = ()
+                -> new RuntimeException("Can't get min value from list: " + numbers);
+        return numbers.stream()
+                .map(s -> Arrays.asList(s.split(",")))
+                .flatMap(Collection::stream)
```

Comment:
.map + .flatMap right after it usually can be replaced with just .flatMap
```suggestion
                .flatMap(s -> Arrays.stream(s.split(",")))
```